cmd/gitserver/server/server.go

```diff
@@ -187,6 +186,10 @@ func NewCloneQueue(jobs *list.List) *cloneQueue {
 
 // Server is a gitserver server.
 type Server struct {
+	// log is a standardized, strongly-typed, and structured logging interface
+	// Production output from this logger (SRC_LOG_FORMAT=json) complies with the OpenTelemetry log data model
+	Log log.Logger
```

Comment:
Logger is a more idiomatic name, that we're using everywhere throughout the codebase with the new package.

For other reviewers: ideally, we should not export it, there is no reason for another package to call a server's scoped logger. But we can't do that because of `internal/vcs/git` who is instantiating servers manually which is out of the scope of this PR. 


```suggestion
	Logger log.Logger
```